src/configurations/destinations/google_adwords_remarketing_lists/db-config.json

```diff
@@ -37,10 +37,10 @@
         "isHashRequired",
         "typeOfList",
         "userDataConsent",
```

Comment:
Is the migration script for audienceId already available in production ?

Reply:
No KC, we need to do hotfix